core/src/style/pointStyleBuilder.cpp

```diff
@@ -310,15 +310,15 @@ bool PointStyleBuilder::evalSizeParam(const DrawRule& _rule, Parameters& _params
     glm::vec2 spriteSize(NAN);
 
     if (_texture) {
-        spriteSize = glm::vec2{_texture->getWidth(), _texture->getHeight()} * _texture->invDensity();
+        spriteSize = glm::vec2{_texture->getWidth(), _texture->getHeight()} / _texture->getDensity();
```

Comment:
Feel this will be called enough times, that it makes sense to save some cycles and not do the division here.

Reply:
TextureOptions now stores a `displayScale` property instead of density so it can be applied with multiplication.